index.js

```diff
@@ -104,15 +104,27 @@ export const buyProduct = (sku) => Platform.select({
   android: () => RNIapModule.buyItemByType(ANDROID_ITEM_TYPE_IAP, sku, null)
 })();
 
+/**
+ * Buy a product with a specified quantity (iOS only)
+ *   Call finishTransaction after receipt validation process.
+ * @param {string} sku The product's sku/ID
+ * @param {number} quantity The amount of product to buy
+ * @returns {Promise<ProductPurchase>}
+ */
+export const buyProductWithQuantity = (sku, quantity) => Platform.select({
+  ios: () => RNIapIos.buyProductWithQuantity(sku, quantity),
+  android: () => RNIapModule.buyItemByType(ANDROID_ITEM_TYPE_IAP, sku, null)
+})();
 
 /**
  * Buy a product without transaction finish (iOS only)
  *   Call finishTransaction after receipt validation process.
  * @param {string} sku The product's sku/ID
+ * @param {number} quantity The amount of product to buy (optional, default = 1)
  * @returns {Promise<ProductPurchase>}
  */
-export const buyProductWithoutFinishTransaction = (sku) => Platform.select({
-  ios: () => RNIapIos.buyProductWithoutAutoConfirm(sku),
+export const buyProductWithoutFinishTransaction = (sku, quantity = 1) => Platform.select({
```

Comment:
Also, I wish not to control the codes that affect both platforms. Hope this to be rollbacked and if you really want this, I think it'd be better to make different method.

Reply:
I'm not sure I understand your comment, `buyProductWithoutFinishTransaction` is already documented as iOS only

Reply:
@lefoy Yes, that may confuse you as I described above. However, that actually works with `android` however. Therefore, I thought it'd be better not to modify codes in `buyProductWithoutFinishTransaction`. I just wanted your feature to be completely dependent to prevent code and features tangling as this module keeps digesting new features.